allure-report-face/src/components/attachment/AttachmentView.hbs

```diff
@@ -38,6 +38,13 @@
     <div class="pane__section">
         <object class="attachment__embed" type="image/svg+xml" data="{{sourceUrl}}">Your browser does not support SVG</object>
     </div>
+{{else if (eq type "video")}}
+    <div class="pane__section">
```

Comment:
It will look better if you would use here the same markup as is used for images.

```
<div class="pane__section attachment__image-container">
    <video controls class="attachment__image">
        <source src="{{sourceUrl}}" type="{{attachment.type}}">
           Your browser does not support video tag
    </video>
</div>
```

That gives some features, like video alignment and prevent overflow if a video is so big.

Also, in that case, `attachment__image` and `attachment__image-container` classes should be renamed to `attachment__image` and `attachment__media-container`respectively for better semantics